pkg/virt-controller/services/template.go

```diff
@@ -62,6 +70,79 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VM) (*kubev1.Pod, error) {
 	return &pod, nil
 }
 
+func (t *templateService) RenderMigrationJob(vm *v1.VM, sourceNode *kubev1.Node, targetNode *kubev1.Node) (*batch.Job, error) {
+	srcAddr := ""
+	dstAddr := ""
+	for _, addr := range sourceNode.Status.Addresses {
```

Comment:
I don't insist on mine, since the list is obviously always very short, so if you think yours is better readable, you can keep it ...